app/views/admin/judges/_form.html.erb

```diff
@@ -16,11 +16,18 @@
         <%= f.text_field :last_name, class: 'form-control' %>
       </div>
     </div>
+
+    <div class='row form-group'>
+      <%= f.label :hidden, 'Hidden', class: 'col-md-2 control-label required' %>
+      <div class='col-md-6'>
+        <%= f.check_box :hidden %>
+      </div>
+    </div>
   </div>
 
   <div class='row'>
     <div class='col-md-1 col-md-offset-7'>
       <%= f.submit 'Save', class: 'btn btn-primary' %>
     </div>
   </div>
-<% end %>
\ No newline at end of file
+<% end %>
```

Comment:
My editor did this automatically. It's good form to have a newline at the end, so I kept it.